lib/babe/build_test.go

```diff
@@ -114,12 +114,12 @@ func createTestBlock(t *testing.T, babesession *Session, exts [][]byte) (*types.
 	}
 
 	// build block
-	block, err := babesession.buildBlock(parentHeader, slot)
-	if err != nil {
-		t.Fatal(err)
+	for {
```

Comment:
perhaps we could do a loop 5x with a delay of 100ms ? ... a infinite loop seems like a potential deadlock....

Reply:
okay I'll put a limit